src/PAModelTests/RoundtripTests.cs

```diff
@@ -13,37 +13,21 @@ namespace PAModelTests
     public class RoundtripTests
     {
         // Apps live in the "Apps" folder, and should have a build action of "Copy to output"
-        [DataTestMethod]
-        [DataRow("MyWeather.msapp")]
-        [DataRow("Chess_for_Power_Apps_v1.03.msapp")]
-        [DataRow("AppWithLabel.msapp")]
-        [DataRow("GalleryTestApp.msapp")]
-        [DataRow("AccountPlanReviewerMaster.msapp")]
-        [DataRow("Marc2PowerPlatformDevOpsAlm.msapp")]
-        [DataRow("SimpleScopeVariables.msapp")]
-        [DataRow("WadlConnector.msapp")]
-        [DataRow("GroupControlTest.msapp")]
-        [DataRow("EmptyTestCase.msapp")]
-        [DataRow("ComponentTest.msapp")]
-        [DataRow("autolayouttest.msapp")]
-        [DataRow("TestStudio_Test.msapp")]
-        [DataRow("ComponentDefinitionWithAllowGlobalAccessProperty.msapp")]
-        [DataRow("ComponentDefinitionWithoutAllowGlobalAccessProperty.msapp")]
-        [DataRow("DuplicateScreen.msapp")]
-        [DataRow("ComponentNameCollision.msapp")]
-        [DataRow("LocaleBasedApplication.msapp")]
-        public void StressTestApps(string filename)
+        [TestMethod]
+        public void StressTestApps()
         {
-            var root = Path.Combine(Environment.CurrentDirectory, "Apps", filename);
+            var directory = Path.Combine(Environment.CurrentDirectory, "Apps");
+            foreach (var root in Directory.GetFiles(directory))
+            { 
+                Assert.IsTrue(File.Exists(root));
```

Comment:
I'd suggest putting the contents of the foreach in a try\catch where the catch is an `Assert.Fail(ex.ToString())` so you can loop through all even when one of the cases throws